### PR TITLE
Introduce discord notification action

### PR DESCRIPTION
**Pull request by KaiVolland**
## Description
This adds the `Discord notification 📯` action to send a message to the discord server every time the package is released.

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

### CODE REVIEW

File: .github/workflows/notify-discord.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  discord-notification:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Discord notification 📯
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        uses: Ilshidur/action-discord@0.3.2
+        with:
+          args: 'Version ${{ github.event.release.tag_name }} of the {{ github.event.repository.name }} has been released.'
```
Comment (jansule) [inline .github/workflows/notify-discord.yml:16]:
```suggestion
          args: '${{ github.event.repository.name }} ${{ github.event.release.tag_name }} has been released.'
```

**Comment by jansule** [comment]
Thanks @KaiVolland. Just 20 more repos to go :grinning:

**Comment by KaiVolland** [comment]
I think @marcjansen had a smart solution for this? Was it a shell script?

**Pull request merged**